src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/index.js

```diff
@@ -103,6 +103,7 @@ IncidentSplitForm.propTypes = {
   }).isRequired,
   subcategories: PropTypes.array.isRequired,
   directingDepartments: directingDepartmentsType.isRequired,
+  isSaving: PropTypes.bool.isRequired,
```

Comment:
Minor point: I'd use `isSubmitting` instead of `isSaving` to keep the relation with the naming of the form consistent (`onSubmit`).